### ISSUE TITLE
Staging Review finding: Button is mis-identified to screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#86832]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** VYE Team
**Product:** Verify Your Enrollment
**Feature:** Verify Your Enrollment

## Findings details
**VA.gov Experience Standard - issue:** [User can't identify an element or its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/mgib-enrollments/benefits-profile/](https://staging.va.gov/education/verify-school-enrollment/mgib-enrollments/benefits-profile/) when editing contact information, the Cancel button has an `aria-label` that misidentifies its purpose. Although the interaction is for contact information, the button is announced to screen reader users as "cancel updating your bank information for GI Bill benefits."

<details><summary>Link, screenshot or steps to recreate</summary>

Here's that button group as currently coded:
```
<div class="button-container">
   <button class="usa-button-primary vads-u-margin-top--0 address-submit-btn-auto-width" aria-label="save your Mailing address for GI Bill benefits" type="submit" aria-live="polite">Save</button>
   <va-button text="Cancel" secondary="" label="cancel updating your bank information for GI Bill benefits" data-qa="cancel-button" data-testid="GI-Bill-Chapters-form-cancel-button" uswds="" class="hydrated"></va-button>
</div>
```

</details>

## Recommended action

Guessing this was a copy/paste error, but make sure the `aria-label` always matches the actual purpose of the element.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jsimonVA** [comment]
aria-label has been updated
https://github.com/department-of-veterans-affairs/vets-website/pull/30534

**Comment by jsimonVA** [comment]
Update has been made and is in staging.
